src/schemas/hazardous-waste-consignment.js

```diff
@@ -0,0 +1,71 @@
+import Joi from 'joi'
+import { isValidHazardousEwcCode } from '../common/constants/ewc-codes.js'
+
+export const NO_CONSIGNMENT_REASONS = [
+  'Non-Hazardous Waste Transfer',
+  'Carrier did not provide documentation',
+  'Household Waste Recycling Centre Receipt'
+]
+
+// Consignment note code formats
+const EA_NRW_PATTERN = /^[A-Za-z]{2,}\/[A-Za-z0-9]{5}[A-Za-z]?$/
```

Comment:
Can we add comments explaining what these patterns match and provide examples

Reply:
added example codes in the comments